### PR TITLE
Spanish Translation: magic link emails for login 

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1057

### CODE REVIEW

File: js/shared.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ const signInFlowRender = (signInEmail) => {
 };
 
 export const sendEmailLink = () => {
+    const preferredLanguage = window.localStorage.getItem("preferredLanguage");
```
Comment (anthonypetersen) [inline js/shared.js:150]: please use the function that has already been created to get preferred language

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
see comment

File: js/shared.js (hunk 1 above)
Comment (jhflorey) [reply js/shared.js]: Solved

File: js/shared.js [hunk 2]
```diff
@@ -147,6 +147,11 @@ const signInFlowRender = (signInEmail) => {
 };
 
 export const sendEmailLink = () => {
+    //Check for language storage
+    let preferredLanguage = window.localStorage.getItem('preferredLanguage');
```
Comment (anthonypetersen) [inline js/shared.js:151]: sorry, I was confused from what you sent... you should be able to reference `getSelectedLanguage()` within `shared.js`

**Review by anthonypetersen — CHANGES_REQUESTED** [review]

File: js/shared.js (hunk 2 above)
Comment (jhflorey) [reply js/shared.js]: Solved

**Review by Gbarra9 — APPROVED** [review]

**Review by we-ai — APPROVED** [review]

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**